cves/CVE-2009-3095.yml

```diff
@@ -54,7 +57,7 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: 404947824a978c89589d0fa59ada0ee1531bb147
-     note:
+     note: 'A check was added to throw out strings with special characters'
    - commit:
      note:
 vccs:
```

Comment:
I would recommend adding a note that says what was changed in commit a83b21763cd2fbfd97bf5cbb5f01ea4d7e7ac3c3.